src/h5web/visualizations/line/models.ts

```diff
@@ -0,0 +1,4 @@
+export enum Glyph {
+  Line = 'Line',
+  Square = 'Square',
+}
```

Comment:
Named `Glyph` to follow [VX's naming](https://vx-demo.now.sh/glyphs). Other possible names include: `Marker`, `DataSymbol`, `Symbol`...